server/controllers/card.js

```diff
@@ -0,0 +1,77 @@
+const asyncHandler = require("express-async-handler");
+const Card = require("../models/Card");
+const { v4: uuidv4, v4 } = require("uuid");
+
+exports.getCards = asyncHandler(async (req, res) => {
+  const Cards = await Card.find();
+
+  return res.json(Cards);
+});
+
+exports.getCard = asyncHandler(async (req, res) => {
+  try {
+    const Card = await Card.findById(req.params.id);
+
+    if (!Card) res.status(404).json({ error: "Card not found" });
+
+    return res.status(200).json(Card);
+  } catch (error) {
+    return res.json(error);
+  }
+});
```

Comment:
Async handler will already handle thrown errors, there is no need to add a `try/catch` inside this handler.